app/views/users/index.html.erb

```diff
@@ -6,6 +6,16 @@
         <span class="profile-link">
           <%= link_to 'See Profile',  user_path(user), class: 'profile-link' %>
         </span>
+
+        <% if current_user.pending_friends.include?(user) %>
```

Comment:
- [ ] There shoud be absolutely NO logic in views. View related logic should be encapsulated in View [helpers](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers).